css/style.css

```diff
@@ -1,11 +1,174 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
+ 
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    position: relative;
+}
+ /* Header  */
+ header {
+    font-size: 24px;
+    width: 100%;
+    height: 20px;
+    padding: 40px;
+    margin: 20px;   
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+}
+
+ .logo {
+    width: 100%;
+    height: 50px;
+    align-self: flex-start;
+}
+
+.navbar {
+ }
+
+.navbar ul {
+    display: flex;
+    flex-flow: nowrap;
+    justify-content: flex-end;
+    align-items: center;
+}
+.navbar ul li {
+    list-style-type: none;
+    margin: 10px;
+    padding: 5px;
+
+}
+.navbar ul li a {
+    /* list-style-type: none; */
+    text-decoration: none;
```

Comment:
It looks like you need to set a `color` rule for these links. They come out purple for me.

Reply:
I fixed this problem.